src/main/java/deepmoblearningbm/DMLBMConfig.java

```diff
@@ -18,11 +18,19 @@ public static class MachineSettings {
         @Config.Comment("Energy cost of Digital Agonizer in RF/t")
         @Config.RangeInt(min = 0, max = 10000)
         public int DIGITAL_AGONIZER_RF_COST = 256;
+        @Config.Name("Digital Agonizer RF Capacity")
+        @Config.Comment("Energy capacity of Digital Agonizer in RF")
+        @Config.RangeInt(min = 0)
+        public int DIGITAL_AGONIZER_RF_CAPACITY = 2147483647;
+        @Config.Name("Digital Agonizer max input RF/t")
+        @Config.Comment("Maximum RF/t Digital Agonizer can accept")
+        @Config.RangeInt(min = 0)
+        public int DIGITAL_AGONIZER_RF_INPUTMAX = 2147483647;
```

Comment:
This configuration doesn't seem to serve any purpose, other than to completely forbid certain models which need too much RF/t to work?

Reply:
removed this config so its now just MAXINT always